app/src/main/res/values/strings.xml

```diff
@@ -5,7 +5,14 @@
     <string name="title_bottom_nav_tasks">Tasks</string>
     <string name="title_bottom_nav_plants">Plants</string>
     
-    <!-- Fragment Plant Detail -->
-    <string name="msg_watering_frequency_text">Watering frequency:</string>
-    <string name="msg_watering_frequency_value">every %d days</string>
+    <!-- Plant Detail Screen -->
+    <string name="msg_detail_watering_text">Watering frequency:</string>
+    <string name="msg_detail_watering_value">every %d days</string>
```

Comment:
You can combine those 2 strings into one, just add separator Watering frequency: \n every %d days
And use only one textview